app/src/components/common/Table.vue

```diff
@@ -44,21 +61,90 @@
 </template>
 
 <script lang="ts">
-import { TableColumn } from '@/types'
+import { TableColumn, SortOrder, SortRule } from '@/types'
 import { Vue, Component, Prop } from 'vue-property-decorator'
 
-@Component
+@Component({
+  computed: {
+    SortOrder () {
+      return SortOrder
+    }
+  }
```

Comment:
what is this `computed` for?

Reply:
This is here so that I can use SortOrder enum values in the template, however, I could remove it if I put comparisons in the getters, will fix it